QEfficient/transformers/modeling_utils.py

```diff
@@ -88,6 +90,7 @@
 my_architectures = ModelArchitectures(
```

Comment:
can you pls rename this to `qeff_supported_architectures `?

Reply:
done